examples/dreamcast/vmu/vmu_beep/beep.c

```diff
@@ -38,29 +39,39 @@
 
 #include <plx/font.h>
 
+#define VMU_START_EFFECT 0x000065F0
+#define VMU_STOP_EFFECT  0x00000000
 
-static plx_font_t *fnt;
-static plx_fcxt_t *cxt;
+static atomic_bool quit;
```

Comment:
LOL! I agree with you, but I saw KOS not doing it in several places for `static` variables (like the previous above two lines), and since they're auto-initialized to zero by default, thought that was just the style I should follow...

But yeah, I personally always do explicitly initialize globals/statics too.